### ISSUE TITLE
Would you please provide unamalgamed source code?

### ISSUE THREAD

**Issue by lanrorbbin** [cesanta/v7#561]
I would like to go through the source code or maybe some other one may fix some bugs. Real source is much better than thousands lines of code in one file. Thanks.

**Comment by eco747** [comment]
+1 for me

**Comment by cia48621793** [comment]
We need 'unamalgamelization' so that it could be more IDE-friendly and hacking-friendly. Loading 10k LOC is a big performance problem on my MacBook.

**Comment by mmikulicic** [comment]
yeah this is is long overdue. we just provided the unamalgamation script for mongoose. I'd like to do the same for v7, stay tuned.

**Comment by eucpp** [comment]
Is there any progress ?

**Comment by mmikulicic** [comment]
Please use the unamalgam tool from mongoose repo: https://github.com/cesanta/mongoose/tree/master/tools
